src/main/java/au/org/ala/biocache/dto/OccurrenceIndex.java

```diff
@@ -439,12 +439,12 @@ public void setInstitutionUid(String institutionUid) {
         this.institutionUid = institutionUid;
     }
 
-    public String getRaw_institutionCode() {
-        return raw_institutionCode;
+    public String getInstitutionCode() {
```

Comment:
Ditto the above

Reply:
It would also affect any consumers of the search service.

Reply:
Change made in commit https://github.com/AtlasOfLivingAustralia/biocache-service/pull/598/commits/9801972bcf1841ee299ea642629e4d663bb8f9b4